static/js/app.js

```diff
@@ -91,12 +91,28 @@ async function sendForm(formAction, formData) {
 function copyToClipboardAndShowMessage(triggeringEl) {
   const aliasCopyBtn = triggeringEl;
   const previouslyCopiedAlias = document.querySelector(".alias-copied");
+  
   if (previouslyCopiedAlias) {
-    previouslyCopiedAlias.classList.remove("alias-copied");
+    previouslyCopiedAlias.classList.remove("alias-copied", "alias-copied-fadeout");
     previouslyCopiedAlias.title = "Copy alias to clipboard";
   }
+  
   aliasCopyBtn.classList.add("alias-copied");
   aliasCopyBtn.title="Alias copied to clipboard";
+
+  // Countdown timer to fade out the "Copied" message after 5 seconds of being displayed
+  let copyMessageCountdown = 0;
```

Comment:
Using `let` here bc it's being updated by the `setInterval`

Reply:
Could we do this with a CSS animation instead? Something like:

```
.alias-copied-fadeout {
  animation-duration: 5s;
  animation-name: fadeOut;
}

@keyframes fadeOut {
  from: {
    opacity: 100%;
  }

  to {
    opacity: 0%;
  }
}
```
?